src/platform/user/tests/authentication/components/loginActions.unit.spec.js

```diff
@@ -71,7 +66,6 @@ describe('login DOM ', () => {
         </Provider>,
       );
       expect(loginButtons.find('[data-csp="dslogon"]').exists()).to.be.true;
-      expect(loginButtons.find('#create-account').exists()).to.be.false;
```

Comment:
You can remove the first `it` test for `vaoccmobile`

We will go back and fix it later on